auth_backends/strategies.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from social.strategies.django_strategy import DjangoStrategy
+
+
+class CurrentSiteDjangoStrategy(DjangoStrategy):
```

Comment:
Good point. Yes, I think I will move this to ecommerce. Any suggestions for where to put it? ecommerce.oauth.strategies, maybe?